mozilla/blurts-server
# Remember the user's last-selected email

# References: 
Jira: MNTOR-1155
Figma: N/A




# Description

We don't want to leave traces of the user's known email addresses after their session expires, so we only store the index of the email. This does mean that if two different users share the same device, one user's selected email might affect the preselected one for the other. That does not seem like a major deal.

# Screenshot (if applicable)

Not applicable.

# How to test

Select a different email address to view the breaches of. Refresh the page. That email address should still be selected.

# Checklist (Definition of Done)
